src/styles/components/Header.scss

```diff
@@ -0,0 +1,9 @@
+.header {
+  padding: 6rem 0.5rem 0 10rem;
+
+  &__logo {
+    min-width: 12rem !important;
+    position: relative !important;
+    height: 100% !important;
```

Comment:
Do we need `position: relative` here? Also, for images, I usually set a specific height or width (or max-width), then set the other dimension to 100%. `min-width` could lead to undesirable results on small screens.